sklearn/linear_model/_huber.py

```diff
@@ -242,6 +244,15 @@ class HuberRegressor(LinearModel, RegressorMixin, BaseEstimator):
     Linear Regression coefficients: [-1.9221...  7.0226...]
     """
 
+    _parameter_constraints = {
+        "epsilon": [Interval(Real, 1.0, None, closed="neither")],
+        "max_iter": [Interval(Integral, 1, None, closed="left")],
+        "alpha": [Real],
```

Comment:
alpha must be non-negative
```suggestion
        "alpha": [Interval(Real, 0, None, closed="left")],
```

Reply:
This is not specified in the docs. I think it should be stated.

Reply:
You can indeed add that it must be in the range ``[0, inf)`` in the parameter description

Reply:
The same as above, you can add in the documentation the sentence to give the valid interval.